tools/bake.py

```diff
@@ -463,12 +493,13 @@ def perform_bake(self, context):
         prioritize_face = context.scene.bake_prioritize_face
         prioritize_factor = context.scene.bake_face_scale
         uv_overlap_correction = context.scene.bake_uv_overlap_correction
-        margin = 0.01
+        margin = 2/resolution #4 pixels of margin no matter image size.
         quick_compare = context.scene.bake_quick_compare
         optimize_static = context.scene.bake_optimize_static # Separate blendshape geometry into its own mesh, improves performance
         apply_keys = context.scene.bake_apply_keys
+        optimize_solid_materials = context.scene.bake_optimize_solid_materials
 
-        # TODO: Option to seperate by loose parts and bake selected to active
+        # TODO: Option to seperate by loose parts and bake selected to active @989onan says: But why?
```

Comment:
Would revert this... 'selected to active' bakes use blender ray-matching, so instead of relying on the UV map for the projection, it's a more smooth result. Downside (and why I came up with the UV thing in the first place) is that it can have artifacts around sharp corners.

Reply:
Ohhhhh! Okay makes sense thanks